spec/bing_ads_ruby_sdk/preprocessors/order_spec.rb

```diff
@@ -12,33 +11,37 @@ def action
       SpecHelpers.wrapper(:customer_management, "SignupCustomer")
     end
 
-    let(:unordered_params) {{
-      "Account" => {
-        "Name" => "test account",
-        "CurrencyCode" => "EUR",
+    let(:unordered_params) {
```

Comment:
Aren't multi-line `let` blocks supposed to be `do ... end`?

Reply:
I can add a quick PR for that if you don't want to. My Editor has a macro for it

Reply:
man its green, why, oh why would you post style changes?

Reply:
the whole point of the linter is to avoid discussions about this